### PR TITLE
Improve UX with better wording

### LINKED ISSUE #276: Section Messagerie : remplacer le texte en cas de page vide
Pour l'instant, s'il n'existe aucune boite liée à un nom de domaine, on voit le message "This table is empty"
avec un visuel de tableau

=> A remplacer par une phrase plus explicite pour l'utilisateur

FR : Aucune boîte mail n'a été créée avec ce nom de domaine.
EN : No mail box was created with this mail domain.

Visuel à supprimer

### LINKED ISSUE #295: Modale pour retirer un utilisateur d'un groupe : modification de wording
Dans la modale de confirmation pour retirer un membre d'un groupe, il faut modifier les textes actuels.

cf. maquette Figma : 
https://www.figma.com/design/Qn0H9DmBjae6pD6HTjKX3i/R%C3%A9gie?node-id=1264-9980&t=WhfLXGOEWcYykC0x-4

ou les wordings ci-dessous :

**TITRE DE LA MODALE**
FR : remplacer par **Retirer le membre du groupe**
EN : OK tel quel

**LIGNE DE TEXTE**
FR : Remplacer par : "Voulez-vous vraiment retirer ce membre du groupe XXX ?"
(avec un espace avant le point d'interrogation s'il vous plaît 🤓)
EN : OK tel quel

**CTA**
FR : Remplacer par "Retirer du groupe"
EN : Remplacer par "Remove from the group"

### LINKED ISSUE #296: Champs de saisie : retirer les icônes "Crayon"
Retirer les icônes crayons de tous les champs de saisie partout où ils se trouvent sur la droite du champ.

Raison : ce n'est pas le standard habituel pour l'utilisation d'une telle icône (qui est généralement présente comme un CTA pour inviter à faire une modification. Exemple : profil Linkedin)

Endroits où le champ a cette icône : création de groupe, modification de nom de groupe

Image

### LINKED ISSUE #303: [a11y] Supprimer l'alternative du logo Régie dans le header
Toutes les images décoratives (= qui n'apportent aucune information) doivent être **ignorées**.
C'est le cas pour les 3 exemples ci-dessous.

1) Logo Régie dans le header.
Il faut donc lui mettre comme alternative : `alt=""`  (rien entre les guillemets)

2) C'est également le cas des icones de groupes/domaines dans le menu intermédiaire

Image

Ces icônes sont purement décoratives. Il faut leur retirer leurs attributs aria, et leur ajouter une alternative vide pour être ignorées : `alt=""`

3) De même, pour les boutons de tri par ordre alphabétique. Dans l'exemple ci-dessous, on peut conserver l'aria-label du button, mais supprimer l'aria-label du SVG, et le remplacer par une alternative vide `alt=""`.

Image

### LINKED ISSUE #304: Correction mineure de wording sur la page d'erreur
Sur les pages d'erreur (cf. screenshots ci-dessous) : 

En français : remplacer "Ouch !" par "Aïe !"  (ouch c'est de l'anglais... oui je sais, je chipote 😉)
En anglais : remplacer "Ouch !" par "Ouch!" (en anglais on ne met pas d'espace avant les ponctuations)

FR

Image

EN

Image

### PR DESCRIPTION
**Pull request by daproclaima**
## Purpose

Improve UX wording and screen reader navigation

## Proposal
- using more meaningful texts for the mailbox list when no mailbox was created for a mail domain, CTA buttons for groups, and mail domains features
- removing the pencil icon from text inputs
- remove icon aria-label from decorative icons 
- add an `alt=""` and `aria-hidden="true"` to decorative images and icons

### motivation
Screen readers should ignore decorative images. However, if no alt is provided, the image source URL will be read out loud. To have an image ignored by screen readers, it takes setting an `alt=""` to it.

For the SVG icons to be ignored by screen readers, they need to take an `aria-hidden="true"`. However, even browser testing tools like Playwright will ignore them, even though they possess an `aria-label="description"`. `aria-label` attributes can hence be removed. To assert their visibility, it takes using visual testing tools such as chromatic.

Closes #276, closes #295, closes #296, closes #303, closes #304

### CODE REVIEW

File: src/frontend/apps/desk/src/components/Link.tsx [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+
 export interface LinkProps {
   $css?: string;
+  tabIndex?: EnumTabIndex;
 }
 
-export const StyledLink = styled(Link)<LinkProps>`
+export const StyledLink = styled(Link).attrs<LinkProps>(({ tabIndex }) => ({
+  tabIndex: tabIndex ?? EnumTabIndex.keyboardNavigationOn,
+}))`
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/components/Link.tsx:13]: Creating an enum for this check feels overly complex. I suggest improving the API of the LinkProps by using a boolean flag instead, for example:
```
export interface LinkProps {
  $css?: string;
  isKeyboardNavigationOn?: boolean;
}
```
Setting the default value to false ensures that the prop is straightforward to use. Exposing clear and concise props is crucial as it simplifies the component interface.

This approach makes it easier for developers to understand and utilize the component effectively.

**Review by lebaudantoine — COMMENTED** [review]
I might not be the best reviewer for changes related to Styles. However, I did notice a small detail that could be simplified in your props.

File: src/frontend/apps/desk/src/features/header/Header.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const Header = () => {
           </LogoGouv>
           <StyledLink href="/">
             <Box $align="center" $gap="1rem" $direction="row">
-              <Image priority src={IconApplication} alt={t('Régie Logo')} />
+              <Image priority src={IconApplication} alt="" />
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/header/Header.tsx:59]:
```suggestion
              <Image priority src={IconApplication} />
```

File: src/frontend/apps/desk/src/features/language/LanguagePicker.tsx [hunk 3]
```diff
@@ -53,12 +53,12 @@ export const LanguagePicker = () => {
           $gap="0.7rem"
           $align="center"
         >
-          <Image priority src={IconLanguage} alt={t('Language Icon')} />
+          <Image priority src={IconLanguage} alt="" />
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/language/LanguagePicker.tsx:56]:
```suggestion
          <Image priority src={IconLanguage} />
```

File: src/frontend/apps/desk/src/features/mail-domains/components/panel/PanelActions.tsx [hunk 4]
```diff
@@ -40,11 +40,7 @@ export const PanelActions = () => {
         $background={isSortAsc ? colorsTokens()['primary-200'] : 'transparent'}
         $color={colorsTokens()['primary-600']}
       >
-        <IconSort
-          width={30}
-          height={30}
-          aria-label={t('Sort domain names icon')}
-        />
+        <IconSort width={30} height={30} aria-label="" />
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/mail-domains/components/panel/PanelActions.tsx:43]:
```suggestion
        <IconSort width={30} height={30} />
```

File: src/frontend/apps/desk/src/features/menu/MenuItems.tsx [hunk 5]
```diff
@@ -74,7 +74,7 @@ const MenuItem = ({ Icon, label, href, alias }: MenuItemProps) => {
         >
           <Icon
             width="2.375rem"
-            aria-label={t(`{{label}} icon`, { label })}
+            aria-label=""
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/menu/MenuItems.tsx:77]:
```suggestion
```

File: src/frontend/apps/desk/src/features/teams/team-management/components/CardCreateTeam.tsx [hunk 6]
```diff
@@ -40,9 +40,9 @@ export const CardCreateTeam = () => {
       <Box $gap="1rem">
         <Box $align="center">
           <IconGroup
+            aria-label=""
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/teams/team-management/components/CardCreateTeam.tsx:43]:
```suggestion
```

File: src/frontend/apps/desk/src/features/teams/team-management/components/ModalRemoveTeam.tsx [hunk 7]
```diff
@@ -105,7 +105,7 @@ export const ModalRemoveTeam = ({ onClose, team }: ModalRemoveTeamProps) => {
         >
           <IconGroup
             className="p-t"
-            aria-label={t(`Teams icon`)}
+            aria-label=""
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/teams/team-management/components/ModalRemoveTeam.tsx:108]:
```suggestion
```

File: src/frontend/apps/desk/src/features/teams/team-management/components/TeamInfo.tsx [hunk 8]
```diff
@@ -44,9 +44,9 @@ export const TeamInfo = ({ team, currentRole }: TeamInfoProps) => {
         </Box>
         <Box $margin="big" $direction="row" $align="center" $gap="1.5rem">
           <IconGroup
+            aria-label=""
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/teams/team-management/components/TeamInfo.tsx:47]:
```suggestion
```

File: src/frontend/apps/desk/src/features/teams/teams-panel/components/PanelActions.tsx [hunk 9]
```diff
@@ -41,15 +41,15 @@ export const PanelActions = () => {
         $background={isSortAsc ? colorsTokens()['primary-200'] : 'transparent'}
         $color={colorsTokens()['primary-600']}
       >
-        <IconSort width={30} height={30} aria-label={t('Sort teams icon')} />
+        <IconSort width={30} height={30} aria-label="" />
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/teams/teams-panel/components/PanelActions.tsx:44]:
```suggestion
        <IconSort width={30} height={30} />
```

File: src/frontend/apps/desk/src/features/teams/teams-panel/components/PanelActions.tsx [hunk 10]
```diff
@@ -41,15 +41,15 @@ export const PanelActions = () => {
...
       </BoxButton>
       <StyledLink href="/teams/create">
         <BoxButton
           aria-label={t('Add a team')}
           $color={colorsTokens()['primary-600']}
           tabIndex={-1}
         >
-          <IconAdd width={30} height={30} aria-label={t('Add team icon')} />
+          <IconAdd width={30} height={30} aria-label="" />
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/teams/teams-panel/components/PanelActions.tsx:52]:
```suggestion
          <IconAdd width={30} height={30} />
```

File: src/frontend/apps/desk/src/i18n/translations.json
Comment (AntoLC) [inline src/frontend/apps/desk/src/i18n/translations.json:1]: With all the `alt` and  `aria-label` in less, I guess you should have more diff in this file, no ?

Comment (daproclaima) [reply src/frontend/apps/desk/src/i18n/translations.json:1]: Yes, I wasted my rebase and did not fix everything.

File: src/frontend/apps/desk/src/features/header/Header.tsx (hunk 2 above)
Comment (daproclaima) [reply src/frontend/apps/desk/src/features/header/Header.tsx:59]: Thank you @AntoLC. It seems that using an `alt=""` is mandatory.
source: https://www.w3.org/WAI/tutorials/images/decorative/

Also, the Image component from 'next/image' requires it.

File: src/frontend/apps/desk/src/features/teams/member-management/components/MemberGrid.tsx [hunk 11]
```diff
@@ -107,7 +107,7 @@ export const MemberGrid = ({ team, currentRole }: MemberGridProps) => {
       {currentRole !== Role.MEMBER && (
         <Box $margin={{ all: 'big', bottom: 'small' }} $align="flex-end">
           <Button
-            aria-label={t('Add members to the team')}
+            aria-label={t('Add a member to the team')}
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/teams/member-management/components/MemberGrid.tsx:110]: The select to add member is a multi select, you can add more than one members.

**Review by AntoLC — APPROVED** [review]

**Pull request merged**